tests/archives/archive_base_test.py

```diff
@@ -119,6 +119,16 @@ def test_stats_add_and_overwrite():
     assert np.isclose(archive.stats.obj_mean, 3.0)
 
 
+#
+# index_of_single() tests -- on GridArchive only.
+#
+
+
+def test_index_of_single():
+    data = get_archive_data("GridArchive")
+    assert data.archive.index_of_single(data.behavior_values) == data.int_index
```

Comment:
In the future, maybe change behavior_values to measures_batch as well in get_archive_data